CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Next Release
+## 25.2.0
```

Comment:
Should also include a note that this includes the network configuration for the Electra on the holesky and sepolia networks